src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name, health = 100, hidden = false) {
+    this.name = name;
+    this.health = health;
+    this.hidden = hidden;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health = 100, hidden = false) {
+    super(name, health);
+    this.hidden = hidden;
```

Comment:
There is no need to redeclare the 'hidden' property and set it in the constructor of the Herbivore class since it's already being set in the Animal class constructor. However, as per the previous comment, 'hidden' should be removed from the Animal class and kept only in Herbivore.